sdk/extensions/ci.yml

```diff
@@ -0,0 +1,46 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/extensions/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/extensions/
+
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```

Comment:
What about other extensions like Microsoft.Extensions.Azure? Should they also be moved into here?

Reply:
Yes, I'm planning to do that too. Maybe even rename it.